lib/treewalk/splitter.js

```diff
@@ -33,6 +34,11 @@ export class TreewalkCarSplitter {
       channel = val.channel
       if (val.out) yield val.out
     }
+
+    if (!channel) {
+      throw new Error('no blocks in the car')
```

Comment:
This is impossible, `this._cars` will yield a channel or throw. Lets change the error message here so if we ever change the code and it does happen then it's a bit less confusing.

```suggestion
      throw new Error('missing CAR writer channel')
```

Reply:
sounds good the message error. I know that a channel will always exist, but typescript catches this as an error... Other option would be to